tests/store/test_store_api.py

```diff
@@ -223,13 +223,13 @@ def test_update_tower_jobs(analysis_store: MockStore, tower_jobs: List[dict], ca
     assert not analysis.failed_jobs
 
     # WHEN jobs are updated
```

Comment:
maybe? Updated sounds like they are already replaced by new values
```suggestion
    # WHEN jobs are cleaned
```

Reply:
Done